### PR TITLE
chore(weave): Add import time benchmarking script

### PR DESCRIPTION
**Pull request by andrewtruong**
This PR adds a benchmarking script and a test to make sure import times don't exceed 2s.  As of writing, the import time is ~1.3s on an M1 MBP

### CODE REVIEW

**Review by bcsherma — APPROVED** [review]
lgtm

File: tests/trace/test_import.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+def test_import_not_slow(monkeypatch):
+    root_dir = Path.cwd().parent.parent  # root dir of repo
+    monkeypatch.setenv("PYTHONPATH", str(root_dir))
+
+    from scripts.benchmark_import import run_single_import
+
+    import_time = run_single_import()
+
+    # Ideally the import takes < 1s, but in CI it can take up to 3s.
+    assert import_time < 3, f"Import time was {import_time} seconds"
```
Comment (gtarpenning) [inline tests/trace/test_import.py:13]: make configurable? also pr description says 2s

Comment (andrewtruong) [reply tests/trace/test_import.py:13]: Updated description

Comment (gtarpenning) [reply tests/trace/test_import.py:13]: thx

**Pull request merged**